vendor/bat-native-ads/src/bat/ads/internal/features/ad_serving/ad_serving_features.cc

```diff
@@ -14,6 +15,10 @@ namespace {
 
 const char kFeatureName[] = "AdServing";
 
+const char kFieldTrialParameterDefaultAdNotificationsPerHour[] =
+    "default_ad_notifications_per_hour";
+const int kDefaultDefaultAdNotificationsPerHour =
```

Comment:
`kDefaultDefaultAdNotificationsPerHour` should be `kDefaultAdNotificationsPerHour`

Reply:
hm seems like this is intentional (seeing it in tests as well), reads a bit funny :)

Reply:
Unfortunately it is intentional as it is a default value of a default feature value :-(, can we revisit this later sir?